src/ccsr/src/client.rs

```diff
@@ -364,6 +395,65 @@ impl fmt::Display for GetByIdError {
     }
 }
 
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct SubjectConfig {
+    pub compatibility_level: CompatibilityLevel,
+    // There are other fields to include if we need them.
+}
+
+/// Errors for schema lookups by subject.
+#[derive(Debug)]
+pub enum GetSubjectConfigError {
+    /// The requested subject does not exist.
+    SubjectNotFound,
+    /// The compatibility level for the subject has not been set.
+    SubjectLevelCompatibilityNotSet,
```

Comment:
I just realized this probably wants to be `SubjectCompatibilityLevelNotSet`! 😀

Reply:
ah good catch!